libs/blocks/gnav/gnav.css

```diff
@@ -872,6 +916,66 @@ header .gnav-profile.gnav-navitem.has-menu.is-open > a:before {
 
   /* END PROFILE MULTI CLOUD SIGNIN OVERRIDE */
 
+  /* APP LAUNCHER */
+
+  header .app-launcher {
+    box-sizing: border-box;
+    display: flex;
+    grid-area: apps;
+    justify-content: center;
+    position: relative;
+    padding: 0 12px 0 0;
+  }
+
+  header .gnav-navitem.has-menu.is-open .gnav-navitem-menu.app-menu {
+    background-color: var(--color-white);
+    display: flex;
+    position: absolute;
+    left: unset;
+    right: 0;
+    border-radius: 0 0 4px 4px;
+    border-left-color: #cacaca;
+    border-right-color: #cacaca;
+    border-bottom-color: #cacaca;
+    box-shadow: 0 1px 4px rgb(0 0 0 / 15%);
+    width: 334px;
+  }
+
+  header .app-launcher .apps {
+    display: flex;
+    flex-flow: row wrap;
+    max-height: 80vh;
+    overflow-y: scroll;
+  }
+
+  header .app-launcher .apps > li {
+    width: 33.3333%;
+    max-width: 33.3333%;
+    flex: 1 1 auto;
+    min-height: auto;
+  }
+
+  header .app-launcher .apps li > a {
+    display: flex;
+    flex-flow: column;
+    white-space: unset;
+    text-align: center;
+    padding: 16px 0 13px;
+    min-height: 85px;
+    justify-content: space-evenly;
+  }
+
+  header .app-launcher .apps li > a:hover {
+    background-color: #f4f4f4;
```

Comment:
shouldn't this color be in a var?

Reply:
Only using this color in this one spot, so no.

Reply:
ok, let's discuss colors outside of the context of the PR sometime then 🙈 . In my opinion colors should always be in vars, so that it's easier to make changes to the whole color theme if that wish ever comes - but yeah.. all good for me 🙌